### PR TITLE
Start running cube-map-uploads-out-of-order as part of conformance su…

### PR DESCRIPTION
**Pull request by kenrussell**
…ite.

It looks like the underlying driver bug has been fixed, per #932.

### CODE REVIEW

**Comment by kenrussell** [comment]
It looks like the underlying driver bug has been fixed. Time to start running this test more generally.

CC'ing @zhenyao @jdashg @grorg @RafaelCintron .

File: sdk/tests/conformance/textures/misc/cube-map-uploads-out-of-order.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+var color = new Uint8Array([128, 192, 255, 255]);
+for (var ii = 0; ii < 6; ++ii) {
+    gl.texImage2D(gl.TEXTURE_CUBE_MAP_POSITIVE_X + ii, 0, gl.RGBA, 1, 1, 0, gl.RGBA, gl.UNSIGNED_BYTE, color);
+}
+gl.bindTexture(gl.TEXTURE_CUBE_MAP, tex);
+gl.generateMipmap(gl.TEXTURE_CUBE_MAP); // there's no need to call this but the code doesn't check the size.
+
+var textureData = new Uint8Array(dataWidth * dataHeight * 4);
+
+// The first texture has downlaoded
```
Comment (zhenyao) [inline sdk/tests/conformance/textures/misc/cube-map-uploads-out-of-order.html]: downlaoded -> downloaded

**Comment by zhenyao** [comment]
You need to add the test to the 00_* list for it to be run.

**Comment by kenrussell** [comment]
Thanks Mo for your review. I actually added it to 00_test_list.txt locally, but somehow forgot to add that to the pull request.

Review feedback addressed.

**Comment by zhenyao** [comment]
lgtm

**Pull request merged**